Code/GraphMol/DistGeomHelpers/testDgeomHelpers.cpp

```diff
@@ -1727,6 +1727,27 @@ void testEmbedParameters() {
     delete mol;
   }
   {
+      std::string fname =
```

Comment:
This test claims to be using ETKDGv2, but it's using the reference data from ETKDG.
If this is an example where the two versions return exactly the same results, this is a bad test. If they should return different results then maybe you forgot to check in the reference file and update this test?